apps/cli/src/commands/doctor.ts

```diff
@@ -0,0 +1,47 @@
+import { Command, Flags } from '@oclif/core';
+import { execSync } from 'child_process';
+
+export default class DoctorCommand extends Command {
+  static description = 'Verify the minimal requirements for the sunodo execution commands';
+
+  static examples = ["<%= config.bin %> <%= command.id %>"];
+
+  static flags = {
+    help: Flags.help({ char: 'h' }),
+  };
+
+  public async run() {
+    try {
+      // Check Docker version
+      this.log('Checking Docker installation...');
+      const dockerVersionOutput = execSync('docker -v').toString().trim();
```

Comment:
Use `docker version --format json` and parse the output.
We need to check for a minimum version (which I honestly don't know yet)

Reply:
Fixed ! 

We just need to define minimum docker version.